functions/Set-DbaLogin.ps1

```diff
@@ -50,7 +50,10 @@ function Set-DbaLogin {
         Grant access to SQL Server
 
     .PARAMETER PasswordPolicyEnforced
-        Should the password policy be enforced.
+        Should the password policy be enforced (check_policy).
+
+    .PARAMETER PasswordExpirationEnabled
+        Should the password expiration check be enforced (check_expiration). In order to enable this option the PasswordPolicyEnforced (check_policy) must also be enabled for the login.
```

Comment:
Do we need to validate this before trying to enable it? Or depend on the error that SQL Server will return?

Reply:
Yeah, good catch. I was originally thinking to allow the SQL error to propagate back to the caller, but that would mean the alter() would generate an exception and stop processing other logins. I've added in some validation and an error message to help the caller know what they can do to correct the error.